pages/[resourceType]/[id]/index.tsx

```diff
@@ -16,6 +17,7 @@ function ResourceIDPage() {
   const [fetchingError, setFetchingError] = useState(false);
   const [loadingRequest, setLoadingRequest] = useState(false);
   const [pageBody, setPageBody] = useState("");
+  const [evaluateMeasureButton, setEvaluateMeasureButton] = useState(<></>);
```

Comment:
I don't think we need this state variable at all (though obviously it does work this way).

We already have access to the resourceType through the router query. So theoretically when we go to render the buttons, we can just do a conditional render rather than setting a state variable (more below)